modules/api/sync/package/index.py

```diff
@@ -0,0 +1,44 @@
+import psycopg2
+import logging
+import datetime
+import csv
+import decree_provider
+import decree_repository
+import restriction_provider
+import restriction_repository
+import zone_repository
+
+def lambda_handler(_event, _context):
+    connection = psycopg2.connect(database="ecowater", user="ecowater", password="ecowater", host="pg", port="5432")
```

Comment:
to do variables d'environement

Reply:
J'ai une créé une layer pour la partie connection BDD. Je pourrais rajouter le code pour cette partie.